src/data/factored_vocab.cpp

```diff
@@ -655,6 +655,39 @@ std::string FactoredVocab::surfaceForm(const Words& sentence) const /*override f
   return res;
 }
 
+size_t FactoredVocab::getTotalFactorCount() const {
+  return factorVocabSize() - groupRanges_[0].second;
+}
```

Comment:
Auxiliary function that return the total number of factors (no lemmas) in a factored vocabulary.

Reply:
Please add this to the code as function documentation. Preferably use the doxygen syntax that we've established in the PR with documentation.

Reply:
comments added